src/libraries/System.Text.Json/src/System/Text/Json/Serialization/JsonSerializer.Read.Utf8JsonReader.cs

```diff
@@ -105,6 +109,10 @@ public static TValue Deserialize<TValue>(ref Utf8JsonReader reader, JsonSerializ
         /// <exception cref="ArgumentException">
         ///   <paramref name="reader"/> is using unsupported options.
         /// </exception>
+        /// <exception cref="NotSupportedException">
```

Comment:
Reminder: Port these to api docs so they become visible on the docs site:
https://github.com/dotnet/dotnet-api-docs/blob/891ff666a6fb22862cc9fa1b8f868c8725e9bd8c/xml/System.Text.Json/JsonSerializer.xml#L172-L185

https://docs.microsoft.com/en-us/dotnet/api/system.text.json.jsonserializer.deserialize?view=netcore-3.1#System_Text_Json_JsonSerializer_Deserialize__1_System_Text_Json_Utf8JsonReader__System_Text_Json_JsonSerializerOptions_